gtsam/gtsam.i

```diff
@@ -2166,6 +2166,31 @@ virtual class NoiseModelFactor: gtsam::NonlinearFactor {
   Vector whitenedError(const gtsam::Values& x) const;
 };
 
+#include <gtsam/nonlinear/CustomFactor.h>
+virtual class CustomFactor: gtsam::NoiseModelFactor {
+  // Note CustomFactor will not be wrapped for MATLAB, as there is no supporting machinery there.
```

Comment:
But, it's still in `gtsam.i`. Explain (in comment) how the wrapper code knows to not wrap this and make sure MATLAB wrapper still works.

Reply:
done